src/components/IOUConfirmationList.js

```diff
@@ -274,8 +279,14 @@ class IOUConfirmationList extends Component {
                     </View>
                 </ScrollView>
                 <FixedFooter>
+                    {this.props.network.isOffline && (
+                        <Text style={[styles.formError, styles.pb2]}>
+                            {this.props.translate('session.offlineMessage')}
```

Comment:
I would suggest a new message here, as there is nothing for the user to 'Retry'... it suggests that the user took an action. Could we add a new String with just the first half of the message: 
`'Looks like you\'re offline. Please check your connection and try again.'`